device/accton/x86_64-accton_as7116_54x-r0/installer.conf

```diff
@@ -1,4 +1,5 @@
 CONSOLE_PORT=0x3f8
 CONSOLE_DEV=0
 CONSOLE_SPEED=115200
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="pci=noaer"
\ No newline at end of file
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="pci=noaer"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="tg3.short_preamble=1 tg3.bcm5718s_reset=1"
```

Comment:
should you just have one ONIE_PLATFORM_EXTRA_CMDLINE_LINUX? Are they conflicting to each other?

Reply:
@lguohan, you are right. This platform is not even maintained by Accton. 
I should restore it back.